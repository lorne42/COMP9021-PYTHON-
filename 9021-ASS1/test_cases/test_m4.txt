"At least one of Sir John and Sir Dennis is a Knave," Sir Ian said.
Sir Ishmael claims: "at most one of Sir John and I is a Knight."
Sir Sirius said: "exactly one of Sir Ian, Sir Alice, Sir Bob and Sir Carol is a Knight."
Sir Bob said: "all of us are Knaves."
"At least one of Sir Alice and Sir Dennis is a Knight," whispered Sir Bob.
"I am a Knight," whispered Sir Eleonore.
"Sir Ishmael is a Knave," whispered Sir Sirius.
Sir Frank said: "Sir Greg or Sir Carol is a Knight."
Sir Greg said: "Sir Julia, I, Sir Dennis or Sir Carol is a Knight."
Sir Julia said: "Sir Eleonore and Sir Frank are Knaves."