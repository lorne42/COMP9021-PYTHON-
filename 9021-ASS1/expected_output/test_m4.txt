The Sirs are: Alice Bob Carol Dennis Eleonore Frank Greg Ian Ishmael John Julia Sirius
There are 2 solutions.